csm_web/frontend/static/frontend/css/style.css

```diff
@@ -196,9 +196,12 @@ header {
 	color: #575757;
 }
 
+.modal-confirmation-container {
+	display: flex;
+}
+
```

Comment:
Ideally, your changes to `style.css` should instead be put into its own CSs file, specifically for affinity sections. This is perhaps less relevant to the components you're editing, but for things like the manage labels modal, new CSS styles should always be in their own files, to keep things separate and modular.